### PR TITLE
add kind to Learning environment Ecosystem

### LINKED ISSUE #14893: add kind (Kubernetes IN Docker) to the list of learning environment
**This is a Feature Request**

**What would you like to be added**
Add kind project to the list of Docker-based solutions when in learning environment

**Why is this needed**
This document should be updated as always there is new docker based solutions to run kubernetes 

**Page to Update:**
https://kubernetes.io/docs/setup/#learning-environment

### PR DESCRIPTION
**Pull request by medmedchiheb**
Fixes #14893

### CODE REVIEW

**Comment by medmedchiheb** [comment]
@mikedanese what do you think!

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ If you're learning Kubernetes, use the Docker-based solutions: tools supported b
 |                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
 |                     | [k3s](https://k3s.io)|
 |                     | [Ubuntu on LXD](/docs/getting-started-guides/ubuntu/)|
+|                     | [kind k8s IN Docker](https://github.com/kubernetes-sigs/kind)|
```
Comment (sftim) [inline content/en/docs/setup/_index.md]: Couple of comments.
I suggest hyperlinking to https://kind.sigs.k8s.io/
Also, I'd make the link text be “kind” or “KinD” and follow it with “(Kubernetes in Docker)”.

What do you think @medmedchiheb ?

Comment (sftim) [reply content/en/docs/setup/_index.md]: Hold on, I just noticed that Kubernetes IN Docker is already there in the other column.

Comment (medmedchiheb) [reply content/en/docs/setup/_index.md]: ok then, let's revert and change the first one "link and name". i agree with you it's better

**Pull request closed without merging**